std/process.d

```diff
@@ -80,8 +80,7 @@ License:
 Source:
     $(PHOBOSSRC std/_process.d)
 Macros:
-    OBJECTREF=$(D $(LINK2 object.html#$0,$0))
-    LREF=$(D $(LINK2 #.$0,$0))
+    OBJECTREF=$(LINK2 object.html#$0,$(D $0)))
```

Comment:
An `OBJECTREF` looks like this:
![image](https://cloud.githubusercontent.com/assets/4370550/23589662/80d23228-01d1-11e7-8376-8dcd416acb45.png)